frontend/src/components/common/SimpleTable.tsx

```diff
@@ -62,6 +65,7 @@ export interface SimpleTableProps {
   rowsPerPage?: number[];
   emptyMessage?: string;
   errorMessage?: string | null;
+  sortObjSetter? : Dispatch<SetStateAction<{ label: string; isIncreasingOrder: boolean }>>;
```

Comment:
Let's discuss this one on Slack because I don't understand exactly why it's needed, and it seems to add a lot of complexity to the implementation.